it/mail/chrome/messenger/messengercompose/composeMsgs.properties

```diff
@@ -190,229 +190,229 @@ sendAnyway=Send anyway
 
 ## Strings use for the save message dialog shown when the user close a message compose window
-saveDlogTitle=Salvataggio messaggio
-saveDlogMessage=Il messaggio non è stato spedito. Si vuole salvare il messaggio nella cartella Bozze?
+saveDlogTitle=Salva messaggio
+saveDlogMessage=Il messaggio non è stato inviato. Vuoi salvare il messaggio nella cartella Bozze?
 saveDlogSaveBtn=Salva
-saveDlogDontSaveBtn=Don't Save
+saveDlogDontSaveBtn=Non salvare
 saveDlogCancelBtn=Annulla
 
 ## generics string
 defaultSubject=(nessun oggetto)
-chooseFileToAttach=Scelta allegato
+chooseFileToAttach=Allega file
 
 ## LOCALIZATION NOTE (SaveDialogMsg): %1$S is the current subject of the compose window
-windowTitle=Compose: %1$S
+windowTitle=Scrivi: %1$S
 
 ## String used by the dialog that ask the user to enter a subject
 sendMsgTitle=Invia messaggio
-subjectDlogMessage=You did not specify a subject for this message. If you would like to provide one, please type it now.
+subjectDlogMessage=Non hai specificato un oggetto per questo messaggio. Se vuoi inserirne uno, digitalo ora.
 
 ## String used by the dialog that informs the user about the newsgroup recipient
-recipientDlogMessage=Questo account gestisce solo destinatari di posta. Se si continua saranno ignorati i gruppi di discussione.
+recipientDlogMessage=Questo account supporta solo destinatari e-mail. Se continui i newsgroup saranno ignorati.
 
 #String used by the alert that tells the user an e-mail address is invalid
-addressInvalid=%1$S non è un indirizzo e-mail valido in quanto non è nella forma utente@server. Deve essere corretto prima di inviare l'e-mail.
+addressInvalid=%1$S non è un indirizzo e-mail valido poiché non è nella forma utente@server. Devi correggerlo prima di inviare l'e-mail.
 ## String used by the dialog that ask the user to attach a web page
-attachPageDlogTitle=Indicare l'indirizzo da allegare
+attachPageDlogTitle=Indica un percorso da allegare
 attachPageDlogMessage=Pagina web (URL):
 
 ## String used for attachment pretty name, when attachment is a message
 messageAttachmentSafeName=Messaggio allegato
 ## String used for attachment pretty name, when attachment is message part
-partAttachmentSafeName=Parte allegato al messaggio
+partAttachmentSafeName=Allegato parte del messaggio
 
 ## String used by the Initialization Error dialog
 initErrorDlogTitle=Composizione messaggio
 ## LOCALIZATION NOTE (initErrorDlogMessage): %1$S will be replaced by the error string.  do not translate \n
-initErrorDlogMessage=Si è verificato un errore durante la creazione di una finestra per la composizione del messaggio. Riprovare nuovamente. %1$s
+initErrorDlogMessage=Si è verificato un errore durante la creazione di una finestra per la composizione del messaggio. Si prega di riprovare di nuovo. %1$s
```

Comment:
CORRECTIONS NEEDED. It should be as follows: 

+initErrorDlogMessage=Si è verificato un errore durante la creazione di una finestra per la composizione del messaggio. Si prega di riprovare. %1$s